docs/sources/administration/plugin-management/plugin-install.md

```diff
@@ -21,11 +21,24 @@ weight: 120
 
 # Install a plugin
 
-Besides the UI, you can use alternative methods to install a plugin depending on your environment or set-up.
+{{< admonition type="caution" >}}
+
+Plugin installation from the Grafana website will be removed in February 2026. Use the options in this document instead.
+
+{{< /admonition >}}
```

Comment:
I think we need to make it clearer that this warning is just for Grafana Cloud users (who may be visiting this page naturally or via other deprecation notices).   But given this is the OSS doc, i've softened it to `note` and updated the text as my _suggestion_. Additionally, things like CLI/Airgapped don't apply. 

```suggestion
{{< admonition type="note" >}}

Installing plugins from the Grafana website into a Grafana Cloud instance will be removed in February 2026. Use the following guidance to install through the Grafana UI

{{< /admonition >}}
```